viscy/light/data.py

```diff
@@ -318,7 +320,10 @@ class HCSDataModule(LightningDataModule):
     :param bool caching: whether to decompress all the images and cache the result,
         will store in ``/tmp/$SLURM_JOB_ID/`` if available,
         defaults to False
-    :param str ground_truth_masks: path to the ground truth segmentation masks,
+    :param bool normalize_source: whether to normalize the source channel,
```

Comment:
source channels?

Reply:
If this is set to `True` then all source channels will be normalized.